### PR TITLE
Add tests for drag-and-drop uploading.

### PR DESCRIPTION
**Pull request by manthey**
Adjust test code to match our js style guides.

This addresses part of issue #1041.  This only tests the main uploads, not the markdown uploads.

### CODE REVIEW

File: clients/web/test/testUtils.js [hunk 1]
```diff
@@ -970,13 +976,129 @@ girderTest.testUpload = function (uploadItem, needResume, error) {
...
+             suffix: girderTest._uploadSuffix});
+    });
+};
+
+/* Test upload drop events.  The drag and drop events are artificially
+ * constructed, so this isn't a perfect test.  It will exercise the event
+ * callbacks, however.
+ *
+ * @param itemSize: the size of the item to drop.
+ * @param multiple: if a number, try to srop this many items.  Only one item is
```
Comment (zachmullen) [inline clients/web/test/testUtils.js]: drop*

**Comment by zachmullen** [comment]
While we're at it, can we extend or refactor this utility to make it accept the drop target element as a parameter? That'd make it easier to test things like the markdown widget.

**Comment by manthey** [comment]
My initial hope was to reuse this for the markdown widget, but the two widgets are fairly different.  The Upload widget shows a different DOM element when you are hovering over it, versus adding a style to the drop zone, and there are a bunch of other differences.  I was going to extend it to the markdown widget in a separate PR, at which point I'll try to reuse as much of this code as possible.

**Comment by zachmullen** [comment]
Ok, that sounds good. LGTM once travis is green

**Pull request merged**